DOCS/man/vf.rst

```diff
@@ -583,6 +583,10 @@ Available mpv-only filters are:
     ``display_fps``
         Refresh rate of the current display. Note that this value can be 0.
 
+    ``display_res``
+        Resolution of the current display. Note that this will not react
+        to monitor changes and may not work with single image streams.
+
```

Comment:
What's format of this var?
`[aa, bb]` or anything else?

Reply:
It's an int64_t array. I use vapoursynth so I don't know what that actual looks like in practical usage. The types of the other variables aren't documented at all so I assume it doesn't really matter.

Reply:
I mean if users want to use it in vs. They know how to use it quickily.
Here is a example:
```
if display_res.width > 1920 :
    .....
if display_res.height > 1080 :
    .....
```
But now apparently they are not sure about the type of this var.

Reply:
Fair point. I presume the syntax is more like `if display_res[0] > 1920` (or however you subselect an array). I'll elaborate a bit here.